fhttp/http_client.go

```diff
@@ -592,7 +598,8 @@ func NewStdClient(o *HTTPOptions) (*Client, error) {
 			newRemoteAddress := conn.RemoteAddr().String()
 			// No change when it wasn't set before (first time) and when the value isn't actually changing either.
 			if req.RemoteAddr != "" && newRemoteAddress != req.RemoteAddr {
-				log.Infof("[%d] Standard client IP address changed from %s to %s", client.id, req.RemoteAddr, newRemoteAddress)
+				log.S(log.Info, "Standard client IP address changed", log.Attr("thread", client.id), log.Attr("run", client.runID),
+					log.Str("from", req.RemoteAddr), log.Str("new", newRemoteAddress))
```

Comment:
attribute from can be overloaded, should we use a specific attribute. I'm thinking of splunk where you see all the json keys for logs in a time window. Instead of from we can use old. 
For example there is this debug log:
```
log.Debugf("[%d] Closed ok from %v after reading %d bytes", c.id, c.dest, c.size)
```

Reply:
As mentioned in the summary (level info and up), I haven't changed logs below Info because they are only used when reproducing/debugging fortio issues and that's usually human processed

This being said, what do you mean for here?

Reply:
talked in person